README.md

```diff
@@ -1,14 +1,38 @@
-### Salesforcebulk ESB Connector
+# Salesforce Bulk EI Connector
 
-The Salesforcebulk connector allows you to access the Salesforcebulk REST API through WSO2 ESB. SalesforceBulk is a RESTful API
-that is optimal for loading or deleting large sets of data.You can use it to query, insert, update, upsert, or delete a large number of records
-asynchronously by submitting batches that Salesforce processes in the background.
+The Salesforce Bulk [Connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the [Salesforce Bulk REST API](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/asynch_api_intro.htm) through WSO2 Enterprise Integrator (WSO2 EI). SalesforceBulk is a RESTful API that allows you to either quickly load large sets of your organisation’s data into Salesforce or delete large sets of your organisation’s data from Salesforce. You can use SalesforceBulk to query, insert, update, upsert or delete a large number of records asynchronously, by submitting the records in batches. Salesforce can process these batches in the background.
 
-### Build
+## Compatibility
 
-mvn clean install
+| Connector version | Supported Salesforce SOAP API version | Supported WSO2 ESB/EI version |
+| ------------- | ------------- | ------------- |
+| [1.0.6](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.6) | 34.0 | ESB 5.0.0, ESB 4.9.0, EI 6.3.0, EI 6.4.0    |
+| [1.0.5](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.5) | 34.0 | ESB 4.9.0, 5.0.0, 6.1.0, 6.1.1 |
+| [1.0.4](https://github.com/wso2-extensions/esb-connector-salesforcebulk/tree/org.wso2.carbon.connector.salesforcebulk-1.0.4) | 34.0 | ESB 4.9.0, 5.0.0  |
 
-### How You Can Contribute
-You can create a third party connector and publish in WSO2 Store.
+## Getting started
 
-https://docs.wso2.com/display/ESBCONNECTORS/Creating+and+Publishing+a+Third+Party+Connector
\ No newline at end of file
+#### Download and install the connector
+
+1. Download the connector from [WSO2 Store](https://store.wso2.com/store/assets/esbconnector/details/3957b581-86e0-4909-ab75-c3cfa2ce6494) by clicking the Download Connector button.
```

Comment:
Download the connector from the [WSO2 Store](https://store.wso2.com/store/assets/esbconnector/details/3957b581-86e0-4909-ab75-c3cfa2ce6494) by clicking the Download Connector button.

Reply:
Fixed in a0448a990213d3cdb3f3ae158ef2307753bad900.